src/components/messages/ThreadSelector/ThreadSelector.scss

```diff
@@ -121,6 +121,13 @@
     gap: 1em;
     width: 100%;
     justify-content: flex-start;
+
+    overflow-y: auto;
+    padding-right: 0.5rem;
+    max-height: 75vh;
+    @media only screen and (min-width: 700px) {
+      max-height: 70vh;
+    }
```

Comment:
Ah, slipped again, since I copied some code from the same file. Maybe we can have another issue for replacing `min-width` everywhere with `max-width` ?